test/debug_info/cmn_blk_dbg_info.f90

```diff
@@ -0,0 +1,25 @@
+!RUN: %flang -gdwarf-4 -S -emit-llvm %s -o - | FileCheck %s
+
+!CHECK: @blk_ = global {{.*}} align 64, !dbg !{{[0-9]+}}, !dbg !{{[0-9]+}}, !dbg !{{[0-9]+}}, !dbg !{{[0-9]+}}, !dbg !{{[0-9]+}}, !dbg !{{[0-9]+}}
```

Comment:
Not clear what this is testing. Don't we have to cross-check this dbg metadata with something in the subroutine?

Reply:
This tests whether there is more than one reference has been linked to global variable. But i will add one more check as you mentioned.

Reply:
Intention is to check whether global common block has 6 metadata reference as in program. Please let me know if it is not sufficient.